test/orchard/xref_test.clj

```diff
@@ -38,4 +41,6 @@
     (is (contains? (into #{} (xref/fn-refs #'map)) #'orchard.xref-test/dummy-fn)))
   (testing "with a symbol"
     (is (= (xref/fn-refs 'orchard.xref-test/dummy-fn) '()))
-    (is (contains? (into #{} (xref/fn-refs #'map)) #'orchard.xref-test/dummy-fn))))
+    (is (contains? (into #{} (xref/fn-refs #'map)) #'orchard.xref-test/dummy-fn)))
+  (testing "with a lambda"
```

Comment:
Not sure if this test has any value, given that `times` now appears in all the other tests.

Reply:
> Not sure if this test has any value, given that times now appears in all the other tests.

Right, please craft a separate example such that it only a lambda dependency, so that it's super clear what we're testing.

i.e. it's not ideal to test many things at the same time.

As in a prior note, please avoid "lambda" terminology, in a deftest I'd suggesting a super clear wording so that we can maintain this effectively.

Finally, you can also rename `times` to something like `fn-dep` so that we emphasize what is being tested.